doc/man3/PKCS12_newpass.pod

```diff
@@ -17,6 +17,16 @@ PKCS12_newpass() changes the password of a PKCS12 structure.
 B<p12> is a pointer to a PKCS12 structure. B<oldpass> is the existing password
 and B<newpass> is the new password.
 
+=head1 NOTES
+
+Both B<oldpass> and B<newpass> are NUL-terminated strings in the UTF-8 encoding.
+If either is not a valid UTF-8 byte sequence, then that one is interpreted
```

Comment:
I was trying to be careful to make it clear that if *one* of them isn't a valid UTF-8 sequence, then only that one is interpreted as ISO8859-1; not both. Other than slightly reducing the clarity of that point (in my highly subjective POV) I'm not sure of intent of the changes you suggest. I don't really object to phrasing it your way, but I'm not sure of the reasoning.